### ISSUE TITLE
🔷 [Separate Marketing & Gateway sites] Launch an Updated Gateway Experience to guide Developers to Build and Discover dApps

### ISSUE THREAD

**Issue by charleslavon** [near/near-discovery#1080]
## What ?
- **High Level Goal:** We aim to separate near.org into distinct Marketing (near.org) and Gateway experiences (e.g. apps.near.org, or perhaps discover.near.org)  that inspires visitors to discover, engage with, and build applications.

- **UX Designs:** 

1. The UI of the gateway will be what is currently hidden in absence of the `?sidebar=true` query param. 
2. The UI of near.org will be the current near.org minus the Create Account/ Sign-In CTAs, and eventually the marketing team will own the design and development of future UI changes.  Additional UI changes to the Nav Menu options are detailed below in `Requirements`. 

## Why ?
**tl; dr;** We designate [something].near.org as the exemplar gateway that showcases the value-add of Decentralized Front End Services (Web Engine, QueryAPI, on-chain UIs), and guides developers to discover, remix, and launch their own dapps. 

- **Background:**
[2024 Vision of near.org as a standalone marketing site](https://docs.google.com/document/d/1yrRkIzdW3kGFnXf8r--f4jk1AtRMHddPF9gIAmVhhEU/edit)
[2023 Rationale for splitting near.org into distinct Marketing and Gateway properties. ](https://docs.google.com/document/d/1IDVLXvur05-G17PK9jlEWesG1i4cVZT96xklt5Cqsu8/)

**Key Results:**
  1. Achieve a B+ CSAT score from (logged-in) gateway visitors
  3. Increase the time spent per pageview metric by 5% for gateway visitors.
  4. Increase return visitor metrics by 20%
  5. The click-through engagement on Recently Viewed dApps surpasses 10% of users who have seen the recently viewed dApp UI. (Otherwise, this proposed feature is not worth further development)

---

## Target Users

1. ~~Token Holders: Our engaged users with a vested interest in the success of the Near ecosystem.  Active participants that want to see a vibrant ecosystem and active community.~~
2. ~~Traders: Users seeking to earn return by seeking portfolio returns engaging with projects within the Near ecosytem. One of the major drivers of activity in Web3.~~
4. Founders of single-chain apps: These founders have been our target segment for current products in market and we have seen traction. Need to get more specific and detailed on the Ideal Customer Profile. 
6. Founders of multi-chain defi apps: Multi-chain product offerings are still very limited yet in-demand from users, particularly outside of (and in-between) the EVM and Cosmos ecosystems. 
7. Founders building wallet apps: Founders with largest opportunities to reach an engaged audience at point of need. Wallets can evolve to be a place of discoverability and value accreation for Near. 
8. ~~Founders building new chains: We have seen some degree of market traction for our DA + FF product. During Q2, we will re-evaluate whether this group will remain a target.~~

## How ?

- **Completion Requirements -> the Definition of Done:**

  1. near.org should be a pure marketing site without CTAs to `Create an Account` or `Sign In`. [Perhaps we repurpose the repo of this POC](https://github.com/near/nearorg_container)
  2. near.org should support redirects to the subdomain gateway such that popular existing links are not broken
  - https://near.org/nearcatalog.near/widget/Index
  - https://near.org/zavodil.near/widget/StakingPools
  - https://near.org/signup
  - https://near.org/signin
  - https://near.org/sandbox
  3. The near.org nav menu should be updated such that 
  - `Platform` and `About` are unchanged
  - the `Sandbox` option is removed from `Develop`
  - the `Discover -> Gateways` page should contain an updated link to the subdomain near.org gateway 
  - the `People` option is removed from `Ecosystem`  
  4. The gateway experience should be dedicated to a new subdomain - [something].near.org.     //TODO @thisisjoshford let's collab on figuring out this domain
  5. How might we structure the above changes such that we can fully test the marketing and gateway UXes on preview environments?

  </p>

- **Suggested Release Milestones**
  1. The above requirements should not be deployed into production near.org until the subdomain gateway and marketing site version of near.org have been tested by and feedback collected from our product marketing, data and design colleagues. 

  </p>

- **Go To Market Needs** TODO

  _(List your suggestion on the marketing, documentation, blog, creative, etc assets which are needed in support of each of the above milestones.)_

1. @thisisjoshford do you think it would be sufficient to use near.social posts to announce the new gateway domain?
2. Two-weeks prior to going live with these changes, we should add a toast message that appears to all near.org wallet-connected users with a link to the post mentioned in #1. 

```[tasklist]
### Tasks
- [ ] https://github.com/near/near-discovery/issues/1056
```

**Comment by charleslavon** [comment]
@thisisjoshford @vanessadespain I've included this in the next sprint but marked it as blocked so we can collab on the requirements  & open questions/decisions before kicking off the dev work.

**Comment by vanessadespain** [comment]
Additional Gateway discussion to be had: 

Is the logged in gateway experience completely gated by login? Can you preview any pages and interaction prompts login? 

If completely gated: 
- We'll need a UX flow to determine the pathway to this experience and a sort of marketing page on why you create an account/what's behind the gate. 

If partially gated: 
- The discover menu could be a single link to the gateway
- OR should components page and gateways page are be behind the Near gateway and the only two links are for Near Catalog and the Near Gateway
- OR we keep all current links and ADD Near Gateway

We'll need strategy for how we get users to the Gateway — entry points from Docs, Near.org Marketing, where else?

**Comment by charleslavon** [comment]
@vanessadespain it shouldn't be completely gated. Rather, it could be similar to the current gateway experience on near.org - one can land on a page that supports wallet-connected functionality without first being logged in like this link which is one of our top most clicked links from the google search results which lead to near.org: https://near.org/nearcatalog.near/widget/Index?id=hot-near-wallet-telegram 

We'll need to continue to support links like the one above, by redirecting them to the to the new subdomain location. 

> Is the logged in gateway experience completely gated by login? Can you preview any pages and interaction prompts login?

**Comment by vanessadespain** [comment]
I think we should probably take this time to do a Design Thinking sprint process, much like we're doing with React on Chain. 

There are a number of assumptions made in this ticket, building on previous work conversations most likely, that we can challenge and redefine to meet our OKRs. One of those being the segmentation of the Logged in vs. Logged out experience and we've heard from Product leadership in 2024 push back on the "Logged in" experience of Near.org. This is the perfect moment to level set, name our assumptions and redefine our goals and process for this stream of work. 

From that design thinking sprint we can then develop a Product Hypothesis document and Product Requirements. 

I'm happy to set up the Design Thinking FigJam for us — that seems like where our next steps should be along with developing a plan for handoff of Near.org to the Near Foundation. 

@thisisjoshford @charleslavon @sarahmills
